src/com/jwetherell/algorithms/numbers/Integers.java

```diff
@@ -5,6 +5,12 @@
 import java.util.Map;
 
 public class Integers {
+    private static final int BILLION = 1000000000;
+    private static final int MILLION = 1000000;
+    private static final int THOUSAND = 1000;
+    private static final int HUNDRED = 100;
+    private static final int TEN = 10;
+    //Larger values represented as constants
 
```

Comment:
These values are set as constants for better readability at the top of the code.